packages/theme-check-common/src/checks/variable-name/index.ts

```diff
@@ -0,0 +1,116 @@
+import {
+  LiquidHtmlNode,
+  LiquidTag,
+  LiquidTagAssign,
+  LiquidTagCapture,
+  NodeTypes,
+} from '@shopify/liquid-html-parser';
+import { LiquidCheckDefinition, SchemaProp, Severity, SourceCodeType } from '../../types';
+
+import { camelCase, kebabCase, snakeCase } from 'lodash';
```

Comment:
May we use `lodash-es`, I've tried it locally and it works as expected :)

```suggestion
import { camelCase, kebabCase, snakeCase } from 'lodash-es';
```